docs/index.rst

```diff
@@ -204,6 +204,7 @@ ar71xx-generic
   - UniFi AP Pro
   - UniFi AP Outdoor
   - UniFi AP Outdoor+
+  - UniFi AC Mesh [#ath10k]_
```

Comment:
The name and the ordering are inconsitent: It should be called UniFi AP AC Mesh and be ordered between the Lite and the Pro.

Reply:
But that device has no "AP" in its Name. Its just called "UniFi AC Mesh". I just checked their productpage.
You are right though about the ordering. Will fix that when i get back home.

Reply:
well, you're partly correct, it is also named UAP-AC-M by the vendor...
but upstream OpenWrt doesn't mention the "AP" so we should change this in master branch and your naming should be ok